store/sqlstore/remote_cluster_store.go

```diff
@@ -42,7 +42,32 @@ func (s sqlRemoteClusterStore) Save(remoteCluster *model.RemoteCluster) (*model.
 		return nil, err
 	}
 
-	if err := s.GetMaster().Insert(remoteCluster); err != nil {
+	query := `INSERT INTO RemoteClusters
```

Comment:
Nit: may be good to coalesce the column names and values into their separate rows.

Reply:
Yeah, I meant to have a single line for column names and their values. But this is okay too.